### PR TITLE
HW_3

### PR DESCRIPTION
**Pull request by Nitablack**
Homework 3: Task1.js, Task2.js, Task3.js

### CODE REVIEW

File: hw_3/Task1.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+console.log('item_6 == ' + typeof item_6 + ', item_6_type == ' + typeof item_6_type);
+
+/* 
+22. Создать переменную item_7 и в ней преобразовать item_6 в String.
+23. Создать переменную item_7_type
+24. Присвоить переменной item_7_type тип переменной item_7
+25. Вывести в консоль тип данных item_7 в виде ——  “item_7 == ”  item_7,  “item_7_type == ”  item_7_type ——  
+*/
+
+const item_7 = 'item_6';
```
Comment (VladimirLQA) [inline hw_3/Task1.js]: Pay attention to the task description - need to convert `item_6` value to string, not assign `string "item_6" to item_7`

**Review by anatoly-karpovich — APPROVED** [review]

File: hw_3/Task3.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/*
+Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную 
+n + nn + nnn, где n не перемножаются, а конкатенируются
+*/
+
+let n = 1;
```
Comment (VladimirLQA) [inline hw_3/Task3.js]: Can add checks for `n` variable value using `if...else` or the ternary operator (if u saw second lection)

**Review by VladimirLQA — CHANGES_REQUESTED** [review]

**Comment by VladimirLQA** [comment]
Please, provide minimal description to PR

File: hw_3/Task1.js (hunk 1 above)
Comment (Nitablack) [reply hw_3/Task1.js]: Done

File: hw_3/Task3.js (hunk 2 above)
Comment (Nitablack) [reply hw_3/Task3.js]: Done

File: hw_3/Task3.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/*
+Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную 
+n + nn + nnn, где n не перемножаются, а конкатенируются
+*/
+
+const n = 10;
+if (n < 1 || n > 9) {
```
Comment (VladimirLQA) [inline hw_3/Task3.js]:
```suggestion
if (typeof n !== 'number' || !Number.isInteger(n) || n < 1 || n > 9) {
```

**Review by VladimirLQA — APPROVED** [review]

**Pull request merged**